### DISCUSSION TITLE
Inconsistent grid snapping behaviour when snapping parameters changed?

### DISCUSSION THREAD

**Discussion by NelkonParker** [xournalpp/xournalpp#3621] [General]
This is on Windows 10 and the 1.1.0 Hotfix.

I have noted somewhat anomalous behaviour when trying to use grid snapping. 

In the attached video, I set the grid size to 2.00 and tolerance to 1.00 and the snapping is as I would expect, and I am unable to move a selected object to a position not on the spacing set in Preferences.

On the other hand, when I set grid size to 1.00 and tolerance to 0.5, I can move a selected object to a position not defined by this grid size.

Am I confused about how snapping is supposed to work, or does this look buggy?

https://user-images.githubusercontent.com/88211140/143492843-7f1e1e58-230a-4c81-be70-1ee196de6a47.mp4

**Comment by rolandlo** [comment]
You misunderstood the tolerance. Tolerance 1 means 100% snapping (everything gets snapped). Tolerance 0.5 means 50% snapping (only points that are at most half the maximal distance from the closest grid point get snapped. See also [the website guide](https://xournalpp.github.io/guide/tools/selecsnap/#configuration).
